.github/dependabot.yml

```diff
@@ -1,10 +1,11 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
 version: 2
 updates:
-  - package-ecosystem: "bundler"
+  - package-ecosystem: bundler
     directory: "/"
-    allow:
-      - dependency-name: "rubocop"
-      - dependency-name: "rubocop-performance"
```

Comment:
Removed limitation to rubocop deps. This should allow other deps (mostly transitive deps) to get bumps in our lockfile (which only affects the maintainers, but keeps our lockfile up to date with latest).